src/sentry/preprod/api/endpoints/urls.py

```diff
@@ -55,6 +56,11 @@
 ]
 
 preprod_internal_urlpatterns = [
+    re_path(
+        r"^preprod-artifact/rerun-analysis/$",
```

Comment:
Maybe have this be `rerun-size-analysis` to just better specify it's for size

Reply:
synced offline. Right now this action essentially dispatches a kafka message as if it's a new build, so all applicable products (size / snapshots / etc) would be rerun. in the future, we'll add a `product` field to the kafka message so that we can specify if we want to rerun a specific product for a build, not just "all applicable"